tests/lib/rules/no-warning-comments.js

```diff
@@ -24,15 +24,10 @@ ruleTester.run("no-warning-comments", rule, {
         { code: "// any comment", options: [{ terms: ["fixme", "todo"] }] },
         "// any comment",
         { code: "// any comment", options: [{ location: "anywhere" }] },
-        { code: "// any comment with TODO, FIXME or XXX", options: [{ location: "start" }] },
```

Comment:
This should still be valid: the rule option `{ location: "start" }` means only flag the comment if it starts with TODO/FIXME/XXX.